src/pages/ErrorPage.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Typography, Button } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import { withRouter } from 'react-router-dom';
+
+import Navbar from '../components/Header/Navbar';
+import Footer from '../components/Footer/index';
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1
+  },
+  bodyCopy: {
+    margin: '10rem',
+    textAlign: 'center'
+  },
+  link: {
+    color: theme.typography.h6.color,
+    margin: '1rem'
+  }
+});
+
+class ErrorPage extends Component {
+  handleBack = () => {
+    this.props.history.goBack();
+  };
+  render() {
+    const { classes } = this.props;
+    return (
+      <Grid container item xs={12} className={classes.root}>
+        <Navbar />
+        <Grid
+          container
+          item
```

Comment:
Again, why starting with `<Grid container item` and then having another `<Grid container item`? Shouldn't the first one just be `<Grid container` and the child be `<Grid item`? (Yes, you can make the child also a `container` as in `<Grid item xs={12} container`).